tools/config/project.config.ts

```diff
@@ -4,6 +4,21 @@ import {SeedConfig, normalizeDependencies} from './seed.config';
 export class ProjectConfig extends SeedConfig {
   PROJECT_TASKS_DIR = join(process.cwd(), this.TOOLS_DIR, 'tasks', 'project');
 
+  APP_TITLE = 'My Angular2 App';
+
+  // Autoprefixer configuration.
+  BROWSER_LIST = [
+    'ie >= 10',
+    'ie_mob >= 10',
+    'ff >= 30',
+    'chrome >= 34',
+    'safari >= 7',
+    'opera >= 23',
+    'ios >= 7',
+    'android >= 4.4',
+    'bb >= 10'
+  ];
+
   constructor() {
     super();
     // this.APP_TITLE = 'Put name of your app here';
```

Comment:
IMO it is consistent with the conventions the conventions in the rest of the seed. 

i.e. Project overrides Seed variable.

That said, given the confusion around const and the intent of immutability vs. the reality of the implementation I do agree that APP_TITLE conveys the idea that it can't be modified when in reality, it can be but technically we aren't.

const is kind of a mess, in particular if you come from something like Java or .Net